client/src/docs-ui/menu/menu.tsx

```diff
@@ -60,12 +60,12 @@ export class DocsMenu {
                   <docs-version-switch
                     leftOption={{
                       title: "Libraries",
-                      subTitle: "(preview)",
+                      subTitle: "",
```

Comment:
Some candidates are 1) current 2) preferred 3) latest 4) stable 5) blank. My vote is for ‘latest’.

Reply:
If we wanted to be consistent with UI Components, we can use "Latest"

Reply:
+1 for using latest and legacy for now.

Reply:
Thanks! updated to "(latest)"